test/WebJobs.Script.Tests.Shared/TestMetricsTracker.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Azure.Functions.Platform.Metrics.LinuxConsumption;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests
+{
+    public class TestMetricsTracker : ILinuxConsumptionMetricsTracker
```

Comment:
Moved this class from the other project so that it can be used in both tests projects.